.github/workflows/docs-build.yml

```diff
@@ -31,18 +36,24 @@ jobs:
 
   Push-tag:
     needs: [Build]
-    if: github.event_name != 'pull_request' && github.ref_type == 'tag'
+    if: >
+      github.repository_owner == 'pyxem' &&
+      github.event_name != 'pull_request' &&
+      github.ref_type == 'tag'
     permissions:
       # needs write permission to push the docs to gh-pages
       contents: write
     uses: hyperspy/.github/.github/workflows/push_doc.yml@main
     with:
       output_path: ${{ github.ref_name }}
-      # Add a redirect index.html in the root folder pointing to the tag version
-      redirect_to_version: ${{ github.ref_name }}
+      create_redirect: true
```

Comment:
This is still updating the `index.html` file in the root folder and instead of continuing doing that, I would suggest to make a copy to the documentation on each release (this is what `create_stable_copy=True` does) to the `stable` folder and redirect to that version instead. This is more straightforward and allow to use a persistent url to the "stable" documentation without redirect.